<?xml version="1.0" ?>
<sdf version="1.6">
  <world name="default">
    <plugin filename="SampleSystem"
            name="sample_system::SampleSystem">
    </plugin>
    <plugin filename="SampleSystem"
            name="sample_system::SampleSystem2">
    </plugin>
  </world>
</sdf>
